lstchain/reco/tests/test_utils.py

```diff
@@ -21,6 +21,18 @@ def test_camera_to_altaz():
     np.testing.assert_allclose(sky_coords.alt, pointing_alt, rtol=1e-4)
     np.testing.assert_allclose(sky_coords.az, pointing_az, rtol=1e-4)
 
+def test_radec_to_camera():
+    source_ra  = u.Quantity(83.63308333, u.deg, copy=False)
```

Comment:
here to make the test clearer I will rename those pointing_ra, pointing_dec.
You expect the pointing to have a camera coordinates at (0,0) but not the source, only in ON mode!!